### DISCUSSION TITLE
`resource` not deleted when Workflow deleted

### DISCUSSION THREAD

**Discussion by xiaoyu1095** [argoproj/argo-workflows#11482] [Q&A] [answered]
I initiated a rayjob cr through manifest, but the rayjob cr was not deleted when I deleted the workflow cr.

<img width="754" alt="image" src="https://github.com/argoproj/argo-workflows/assets/19495200/ab61f394-9545-41cc-b242-8acee0f81d80">

Is there any reference material available ？  thanks

**Comment by terrytangyuan** [comment, marked as answer]
Set `setOwnerReference: true`: https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-owner-reference.yaml
